report/renderer/category-renderer.js

```diff
@@ -310,13 +310,14 @@ export class CategoryRenderer {
    * in a collapsed state.
    * @param {Exclude<TopLevelClumpId, 'failed'>} clumpId
    * @param {{auditRefs: Array<LH.ReportResult.AuditRef>, description?: string}} clumpOpts
+   * @param {boolean} openByDefault
    * @return {!Element}
    */
-  renderClump(clumpId, {auditRefs, description}) {
+  renderClump(clumpId, {auditRefs, description}, openByDefault = false) {
     const clumpComponent = this.dom.createComponent('clump');
     const clumpElement = this.dom.find('.lh-clump', clumpComponent);
 
-    if (clumpId === 'warning') {
+    if (clumpId === 'warning' || openByDefault) {
```

Comment:
Can we move the `clumpId === 'warning'` check outside this function so we can keep all the logic together.